packages/ibm-products/src/components/AboutModal/AboutModal.stories.jsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corp. 2020, 2024
+ * Copyright IBM Corp. 2024
```

Comment:
I expected this one to retain the first year as it does in ActionBar 🤔

Reply:
updated! was testing in aboutmodal